aten/src/ATen/nn.yaml

```diff
@@ -5,11 +5,6 @@
   scalar_check:
     output: reduction != Reduction::None || self_->dim() == 0
 
-- name: kl_div(Tensor self, Tensor target, int64_t reduction=Reduction::ElementwiseMean)
-  cname: DistKLDivCriterion
```

Comment:
can you delete `DistKLDivCriterion` as well?

Reply:
it's used in legacy nn :(